### ISSUE TITLE
evaluate rally tracker

### ISSUE THREAD

**Issue by arlin** [phylotastic/phylotastic#55]
see #51 for criteria. Add results to the table at: https://github.com/phylotastic/phylotastic.github.io/blob/master/docs/tracker_evaluations.csv

**Comment by dimus** [comment]
I did not look very closely to rally, but I went on  their site and tried to get some idea about their product. Impression that I got -- it is powerful, but not streamlined, looks like it is created as alternative to Jira.

It has a learning curve, the project is rich but not intuitive. So  with chronic lack of time it would require me a very important reason to study Rally, and figure out their ways.  However it is designed for large teams, and may be someone will find it exciting and learn about it more.

Rally is free to a point, so if team grows beyond that point -- it stops being free. They do have someone who contacted me personally when they saw that I registered there. They did try to sell themselves,  but were not super pushy.
